package.json

```
@@ -13,7 +13,9 @@
     "lint:types": "tsc",
     "i18n": "node_modules/i18next-parser/bin/cli.js",
     "i18n:check": "node_modules/i18next-parser/bin/cli.js --fail-on-warnings --fail-on-update",
-    "test": "jest",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
```

```suggestion
    "test": "vitest",
```

I think the watch mode is a useful default for developers, and vitest automatically switches to "run" mode when it's in CI, so can we replace these with just the one command?

---

Ok!